ibis/expr/operations/udf.py

```diff
@@ -51,7 +51,7 @@ class InputType(enum.Enum):
 
 
 @public
-class ScalarUDF(ops.Value):
+class ScalarUDF(ops.Impure):
```

Comment:
I'm going to split out builtin UDFs and actually-user-defined ones, since builtins aren't in general impure.

Reply:
Will do in a follow-up though.